org.sdrangel.SDRangel.json

```
@@ -0,0 +1,525 @@
+          "url" : "https://github.com/libusb/libusb/releases/download/v1.0.23/libusb-1.0.23.tar.bz2",
+          "sha256" : "db11c06e958a82dac52cf3c65cb4dd2c3f339c8a988665110e0d24d19312ad8d"
+        }
+      ]
+    },
+    {
+      "name": "libxml2",
+      "config-opts": [
+        "--with-python=no"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/GNOME/libxml2/archive/v2.9.9.tar.gz",
+          "sha256": "d673f0284cec867ee00872a8152e0c3c09852f17fd9aa93f07579a37534f0bfe"
+        }
+      ]
+    },
+    {
+      "name": "fftw3",
+      "config-opts" : [
+        "--enable-shared",
+        "--disable-static",
+        "--enable-threads",
+        "--enable-float"
+      ],
+      "sources" : [
+        {
+          "type" : "archive",
+          "url" : "http://www.fftw.org/fftw-3.3.8.tar.gz",
+          "md5": "8aac833c943d8e90d51b697b27d4384d"
+        }
+      ]
+    },
+    {
+      "name": "xxd",
+      "build-options": {
+        "env": {
+          "LIBS": "-lm"
+        }
+      },
+      "cleanup": [
+        "/bin/vim*",
+        "/bin/rvim",
+        "/bin/view",
+        "/bin/rview",
+        "/bin/ex",
+        "/share"
+      ],
+      "config-opts": [
+        "--disable-gui",
+        "--disable-gtk3",
+        "--disable-luainterp",
+        "--disable-python3interp",
+        "--disable-xim",
+        "--disable-xsmp",
+        "--disable-xsmp-interact",
+        "--disable-desktop-database-update",
+        "--disable-icon-cache-update",
+        "--disable-gnome-check",
+        "--disable-motif-check",
+        "--disable-athena-check",
+        "--disable-fontset"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/vim/vim.git",
+          "tag": "v8.1.2102",
+          "commit": "d17a57a43330977b8f4eb36f1f7a4a66a7bb26c8"
+        }
+      ]
+    },
+    {
+      "name": "python-cheetah",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/4e/72/e6a7d92279e3551db1b68fd336fd7a6e3d2f2ec742bf486486e6150d77d2/Cheetah3-3.2.4.tar.gz",
+          "sha256": "caabb9c22961a3413ac85cd1e5525ec9ca80daeba6555f4f60802b6c256e252b"
+        }
+      ],
+      "build-commands": [
+        "python setup.py install --prefix=/app --root=/"
+      ]
+    },
+    {
+      "name": "python-mako",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://files.pythonhosted.org/packages/b0/3c/8dcd6883d009f7cae0f3157fb53e9afb05a0d3d33b3db1268ec2e6f4a56b/Mako-1.1.0.tar.gz",
+          "sha256": "a36919599a9b7dc5d86a7a8988f23a9a3a3d083070023bab23d64f7f1d1e0a4b"
+        }
+      ],
+      "build-commands": [
+        "python setup.py install --prefix=/app --root=/"
+      ]
+    },
+    {
+      "name": "opencv",
+      "buildsystem": "cmake-ninja",
+      "builddir": true,
```

```suggestion
      "builddir": true,
      "build-options": {
          "cxxflags": "-std=c++14"
      },
```

This should fix the opencv build failure.